src/plib/hooks/ContentInclude.php

```diff
@@ -122,16 +118,13 @@ public function getHeadContent()
             return '';
         }
 
-        return '<script src="' . pm_Context::getBaseUrl() . 'domain-connect.js"></script>';
+        return '<script src="' . pm_Context::getBaseUrl() . 'domain-connect.js?v1.1.1"></script>';
     }
 
     public function getJsOnReadyContent()
     {
         return implode("\n", array_map(function($warning) {
-            list($domainId, $message, $closable) = $warning;
-            return "PleskExt.DomainConnect.addConnectionMessage(" .
-                "{$this->jsEscape($domainId)}, {$this->jsEscape($message)}, {$this->jsEscape($closable)}" .
-            ");";
+            return "PleskExt.DomainConnect.addConnectionMessage.apply(null, {$this->jsEscape($warning)});";
```

Comment:
Why not
            return "PleskExt.DomainConnect.addConnectionMessage({$this->jsEscape($warning)});";